tests/integration/StorageBucketTest.php

```diff
@@ -15,7 +15,10 @@ public function setup(): void
 		$dotenv->load();
 		$api_key = getenv('API_KEY');
 		$reference_id = getenv('REFERENCE_ID');
-		$this->client = new  \Supabase\Storage\StorageBucket($api_key, $reference_id);
+		$scheme = 'https';
```

Comment:
After the __constructor is updated none of these should be needed - please remove them.